assets/shaders/custom_material.frag

```diff
@@ -1,11 +1,17 @@
 #version 450
+layout(location = 0) in vec2 v_Uv;
 
 layout(location = 0) out vec4 o_Target;
 
 layout(set = 1, binding = 0) uniform CustomMaterial {
     vec4 Color;
 };
 
+// Naga GLSL does not support the sampler2D type, but only a texture2D + sampler combination
```

Comment:
I think https://github.com/gfx-rs/naga/issues/1012 should be mentioned here

Reply:
In what form ? Add in the comment with the html reference of the naga issue ?

Reply:
yup something like
```suggestion
// Naga GLSL does not support the sampler2D type, but only a texture2D + sampler combination
// See https://github.com/gfx-rs/naga/issues/1012 for details
```